package.json

```diff
@@ -90,6 +90,7 @@
     "stylelint": "^13.3.3",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "typescript": "^3.8.3",
+    "vue-clickaway": "^2.2.2",
```

Comment:
this was missing, causing `npm run docs:build` to fail

Reply:
What was the error? Because this plugin is not used as far as Im aware.

Reply:
it is used in one of the components as a mixin

Reply:
`src/components/VMultiselect/Autocomplete.vue:71`

Reply:
Oh, that makes sense. Yeah, this was another case where I copied over some source code from a different project that I want to keep easily available for whenever I build the component in mind.

Probably not in this next release, but somewhere in the near future I want to create a combobox and possibly an autocomplete component. These already exist in the wild, and I was looking through their source code, but will likely need to change some things. 

Long story short, these components dont actually exist for the library yet. These files are just placeholders.